pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -873,10 +877,18 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
                 player, playerLevel);
 
             // If the user was temp'd, show admins who temp'd the player.
-            if (tempLevel[player] == 1 || tempLevel[player] == 2) {
-                if (Player(playerid)->isAdministrator())
+            {     
+                if (Player(playerid)->isAdministrator() && Player(playerid)->isManagement()) 
+                    {
+                  if (tempLevel[player] == 1 || tempLevel[player] == 2) 
                     format(message, sizeof(message), " %s {CCCCCC}(temp'd by %s){FFFFFF} (Id:%d) - {FF8E02}%s",
-                        Player(player)->nicknameString(), UserTemped[player], player, playerLevel);
+                        Player(player)->nicknameString(), UserTemped[player], player, playerLevel); 
+                    }
+                else {
+                    if (tempLevel[player] == 2) 
+                    format(message, sizeof(message), " %s (Id:%d) - {FF8E02}Temp.Admin",
```

Comment:
Do we really want to display that they're temporary?

If so we'd want to spell it out fully.